app/models/new_msa_component_event.rb

```diff
@@ -26,6 +26,5 @@ def component_is_new
 
   def msa_has_entity_id
     errors.add(:entity_id, 'id is required for MSA component') unless entity_id.present?
```

Comment:
I think it better to have
`errors.add(:entity_id, 'entity_id is required for MSA component')`